dissect/target/target.py

```diff
@@ -256,39 +257,34 @@ def _find(find_path: Path):
             if find_path.is_dir():
                 yield from find_path.iterdir()
 
-        loaded = False
-
         # Treat every path as a unique target spec
         for path in paths:
             path, parsed_path = extract_path_info(path)
 
-            found_loader = False
+            at_least_one_loaded = False
+            errors = []
 
             # Search for targets one directory deep
             for entry in _find(path):
-                loader_cls = loader.find_loader(entry, parsed_path=parsed_path)
-
-                if not loader_cls:
-                    continue
-
-                getlogger(entry).debug("Found compatible loader: %s", loader_cls)
-                found_loader = True
-
+                loader_cls = loader.find_loader(entry, parsed_path=parsed_path) or RawLoader
+                getlogger(entry).debug("Attempting to use loader: %s for: %s", loader_cls, entry)
                 for sub_entry in loader_cls.find_all(entry):
                     try:
                         ldr = loader_cls(sub_entry, parsed_path=parsed_path)
                     except Exception as e:
-                        getlogger(sub_entry).error("Failed to initiate loader", exc_info=e)
+                        if not path.is_dir():
+                            getlogger(sub_entry).error("Failed to initiate loader", exc_info=e)
                         continue
 
                     try:
                         # Attempt to load the target using this loader
                         target = cls._load(sub_entry, ldr)
-                        loaded = True
+                        at_least_one_loaded = True
                         yield target
-                    except Exception as e:
-                        getlogger(sub_entry).error("Failed to load target with loader %s", ldr, exc_info=e)
-                        continue
+                        if entry is path:
+                            break
```

Comment:
I'm not sure if we should break in this loop for this condition (`loader_cls.find_all(entry)`), because the loader may still yield one or more sub entries, even if `entry is path`.

Reply:
If entry is path, the entry seems to be loaded by the dirloader, no other entries can be processed then I assume. To clarify things (at least for myself) I am writing some unit tests for this, that might also make it easier to talk about the different scenarios that can happen. As far as I can see these are not yet tested...?

Reply:
Changed, see unit tests for an overview of all possible scenarios.